flopy/modflow/mf.py

```diff
@@ -968,30 +935,28 @@ def load(
             except KeyError:
                 if ml.verbose:
                     print(
-                        "\nWARNING:\n    External file unit {} does not "
-                        "exist in ext_unit_dict.".format(key)
+                        f"\nWARNING:\n    External file unit {key} does not "
+                        "exist in ext_unit_dict."
                     )
 
         # write message indicating packages that were successfully loaded
         if ml.verbose:
             print("")
             print(
-                "   The following {} packages were successfully loaded.".format(
-                    len(files_successfully_loaded)
-                )
+                f"   The following {len(files_successfully_loaded)} packages "
+                "were successfully loaded."
             )
             for fname in files_successfully_loaded:
-                print("      " + os.path.basename(fname))
+                print(f"      {os.path.basename(fname)}")
             if len(files_not_loaded) > 0:
                 print(
-                    "   The following {} packages were not loaded.".format(
-                        len(files_not_loaded)
-                    )
+                    "   The following {len(files_not_loaded)} packages "
```

Comment:
Whoops! This was one of the few manual edits, and is missing an `f`. I'll follow this up within another refactor touching this file probably in the next week or so.